test/test_parse.py

```diff
@@ -62,60 +87,107 @@ def test_read_original_clinvar_variation_2():
+    scv_trait_0: ClinicalAssertionTrait = list(
+        filter(lambda o: isinstance(o, ClinicalAssertionTrait), objects)
+    )[0]
+    assert scv_trait_0.trait_id == "9580"
+    scv_trait_1 = list(
+        filter(lambda o: isinstance(o, ClinicalAssertionTrait), objects)
+    )[1]
+    assert scv_trait_1.trait_id == "9580"
 
-    scv = objects[10]
+    scv = list(filter(lambda o: isinstance(o, ClinicalAssertion), objects))[1]
     assert scv.assertion_id == "2865972"
-    submitter = objects[8]
+    submitter = list(filter(lambda o: isinstance(o, Submitter), objects))[1]
     assert submitter.id == "507826"
     assert submitter.current_name == "Paris Brain Institute, Inserm - ICM"
-    submission = objects[9]
+    submission = list(filter(lambda o: isinstance(o, Submission), objects))[1]
     assert submission.id == "507826"
     assert submission.submission_date == "2020-11-14"
 
+    # Rcv
+    rcv: RcvAccession = list(filter(lambda o: isinstance(o, RcvAccession), objects))[0]
+    assert rcv.id == "RCV000000012"
+    assert rcv.variation_archive_id == "VCV000000002"
+    assert rcv.variation_id == "2"
+    assert rcv.date_last_evaluated is None
+    assert rcv.version == 5
+    assert (
+        rcv.title
+        == "NM_014855.3(AP5Z1):c.80_83delinsTGCTGTAAACTGTAACTGTAAA (p.Arg27_Ile28delinsLeuLeuTer) AND Hereditary spastic paraplegia 48"
+    )
+    assert rcv.trait_set_id == "2"
+    assert rcv.review_status == "criteria provided, single submitter"
+    assert rcv.interpretation == "Pathogenic"
+
 
-def test_read_original_clinvar_variation_634266():
+def test_read_original_clinvar_variation_634266(log_conf):
     """
     Test a Genotype record
     """
     filename = "test/data/original-clinvar-variation-634266.xml"
     with open(filename) as f:
         objects = list(read_clinvar_xml(f))
 
-    assert 22 == len(objects)
-    assert isinstance(objects[0], Variation)
-    assert isinstance(objects[1], Trait)
-    assert isinstance(objects[2], TraitSet)
-    assert isinstance(objects[3], Trait)
-    assert isinstance(objects[4], TraitSet)
-    assert isinstance(objects[5], Trait)
-    assert isinstance(objects[6], TraitSet)
-    assert isinstance(objects[7], Trait)
-    assert isinstance(objects[8], TraitSet)
-
-    assert isinstance(objects[9], Submitter)
-    assert isinstance(objects[10], Submission)
-    assert isinstance(objects[11], ClinicalAssertion)
-
-    assert isinstance(objects[12], Submitter)
-    assert isinstance(objects[13], Submission)
-    assert isinstance(objects[14], ClinicalAssertion)
-
-    assert isinstance(objects[15], Submitter)
-    assert isinstance(objects[16], Submission)
-    assert isinstance(objects[17], ClinicalAssertion)
-
-    assert isinstance(objects[18], Submitter)
-    assert isinstance(objects[19], Submission)
-    assert isinstance(objects[20], ClinicalAssertion)
-    assert isinstance(objects[21], VariationArchive)
+    assert len(objects) == 42
+    expected_types = [
```

Comment:
The ordering is not important for our use case so far. It's possible we'd want it to be in this order (which matches the order of `clinvar-raw` to give coherent foreign keys), but I do agree this clutters up the tests. We could have just one test function which tests the order for a smaller VCV, and then in the other tests just verify the counts and leave out all the order checks. 

I also did something like on line 62 to get out objects using a filter instead of a hardcoded index, which makes it way easier to make changes to the code without breaking unrelated test assertions.